### DISCUSSION TITLE
Suggestion: Range on /inv name

### DISCUSSION THREAD

**Discussion by BeeRando** [Zrips/CMI#2310] [Feature Requests]
**Description of feature/improvement request:**
In the config, add a block range to /inv name

### Detailed explanation of what it should do, when and in what situation: ###
Not let you /inv name, unless you're within x blocks of that player.

### Basic explanation as to why this feature is needed and/or useful to have: ###
Basically, servers, mostly survival servers, could let people do /inv name, for rp, or general gameplay purposes. Faction/town guards would be able to see peoples inventories, if theyre very close. It'd be cool for server events also, not letting people into an area unless their inventory has or doesnt have certain certain items. For example, if a town or faction had an event, like a wedding, they could check peoples inventories for weapons, and not let them enter the building if they have them. It's a very simple feature, but one that could be used in tons of different ways.

**Comment by sandwally** [comment]
Sounds great.
Would also like to see alternate way where a player can send a view "request" of it's own inventory to another player and they can see it for x amount of time.
